src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -748,30 +751,8 @@ private Result ConstructSqlInjectionResult(ContrastLogReader.Context context)
                 }
             };
 
-            result.CodeFlows = new List<CodeFlow>
-            {
-                new CodeFlow
-                {
-                     ThreadFlows = new List<ThreadFlow>
-                     {
-                         new ThreadFlow
-                         {
-                              Locations = context.PropagationEvents
-                         }
-                     }
-                }
-            };
-
             result.CodeFlows[0].ThreadFlows[0].Locations.Add(context.MethodEvent);
 
-            // ** TODO remove this ** /
```

Comment:
>TODO [](start = 18, length = 4)

That's right. The stacks that occur as properties of threadFlowLocations should not be repeated in result.stacks.